packages/expo-router/src/views/Navigator.tsx

```diff
@@ -3,6 +3,8 @@
 
 import { RouterFactory, useNavigationBuilder } from '@react-navigation/native';
 import * as React from 'react';
+import { Fragment } from 'react';
+import { isEdgeToEdge } from 'react-native-is-edge-to-edge';
```

Comment:
maybe needs to be a new dependency @brentvatne would know. Otherwise seems fine

Reply:
`react-native-is-edge-to-edge` [is already a router dependency](https://github.com/expo/expo/blob/39857334a4da51d6015bc05f0c9aeb082962d314/packages/expo-router/package.json#L117)